stdlib/LinearAlgebra/test/cholesky.jl

```diff
@@ -180,6 +180,7 @@ end
                 # Test error bound on linear solver: LAWNS 14, Theorem 2.1
                 # This is a surprisingly loose bound
                 BB = copy(B)
+                out = zero(BB)
```

Comment:
What is this line good for? Seems like you haven't made any changes to the cholesky code.

Reply:
Just me being stupid, sorry